recipe/meta.yaml

```diff
@@ -1,29 +1,27 @@
-{% set version = "58.0.4" %}
+{% set version = "58.1.0" %}
 
 package:
   name: setuptools
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/s/setuptools/setuptools-{{ version }}.tar.gz
-  sha256: f10059f0152e0b7fb6b2edd77bcb1ecd4c9ed7048a826eb2d79f72fd2e6e237b
+  sha256: 5de67252090e08d25f240f07d80310f778a5a46cdcf9ea9855662630ac8547b2
   patches:
     # Modify setuptools to fail if used in conda build (encourage people to add all deps in meta.yaml).
     - patches/0002-disable-downloads-inside-conda-build.patch
     # distutils patches from python-feedstock
     - patches/0019-Use-ranlib-from-env-if-env-variable-is-set.patch
-    - patches/0008-runtime_library_dir_option-Use-1st-word-of-CC-as-com.patch
     - patches/0012-Disable-new-dtags-in-unixccompiler.py.patch
     - patches/0021-Add-d1trimfile-SRC_DIR-to-make-pdbs-more-relocatable.patch
     - patches/0003-Win32-distutils-Add-support-to-cygwinccompiler-for-V.patch
     - patches/0010-Add-support-for-_CONDA_PYTHON_SYSCONFIGDATA_NAME-if-.patch  # [py<=39 and python_impl=="cpython"]
     - patches/0006-Win32-Fixes-for-Windows-GCC-interop-needed-by-RPy2-a.patch
-    - patches/0034-distutils-fix-ldshared.patch
     # distutils patches from pypy3.6-feedstock
     - patches/0035-pypy-distutils-scheme.patch   # [win or py>37]
```

Comment:
@mattip and @isuruf do we still need this patch?

Reply:
I am not sure. It is useful when compilers define `cc` rather than `gcc`, like clang on linux.

Reply:
OK. I'll refresh the patch.